recipes/infer-subc/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "infer-subc" %}
+{% set version = "0.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/infer-subc-{{ version }}.tar.gz
+  sha256: 7ff6222f124d7167c3334e25d0178adc8d0205b3f5f189fb7ebc4e34379959a8
+
+build:
+  entry_points:
+    - infer_subc = infer_subc.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - aicsimageio
+    - aicssegmentation
+    - jupyter
+    - napari
+    - numpy
+    - pandas
+    - python >=3.8
+    - scikit-image
+    - scipy
+
+test:
+  imports:
+    - infer_subc
```

```suggestion
    - infer_subc
    - infer_subc.core.img
    - infer_subc.organelles
    - infer_subc.workflow
```

`infer_subc.__init__` is essentially empty.